### DISCUSSION TITLE
Play store version of godot, mobile c# support to work with.

### DISCUSSION THREAD

**Discussion by Playeroth** [godotengine/godot-proposals#9198] [Editor]
### Describe the project you are working on

This is a call for c# support in the mobile version of godot. I know this probably isnt the right place but i dont know where, i asked in godot community but no answer. I prefer to work with c# because i can use on other softwares, which the mobile version do not have a way to work with c#. Have they mentioned this somewhere else? Not the export support announcement but to work with c# on mobile version.

### Describe the problem or limitation you are having in your project

not related

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

not related

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

not related

### If this enhancement will not be used often, can it be worked around with a few lines of script?

not related

### Is there a reason why this should be core and not an add-on in the asset library?

c# is already a core of pc version, although its a different version.

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.

The .NET SDK (not to be confused with the .NET runtime) doesn't run on Android, so this is technically not feasible.

**Reply by Playeroth** [reply]
sorry, didnt notice i could have made a discussion topic instead.

Ah i see, makes sense. Thanks for the answer.
